File: 6-makeArrayConsecutive.js
function makeArrayConsecutive2(statues) {

    let count = 0;

    //THIS is a thing I just need to memorize... 
    //_____.sort(function(a,b){return a - b})
    //which tells it HOW we want it to sort for us... 
    let sorted = statues.sort(function(a, b){return a - b});
    //console.log(sorted);
    //.sort(function(a,b){return a - b});
    if(sorted.length <= 1){
        return 0;
    }
    else{
    for(i = 0; i<sorted.length - 1; i++){
        count += (sorted[i+1] - sorted[i]) - 1;
      //console.log(count);
    }
    } 
   
    return count;
  
}

File: 6-makeArrayConsecutive.java
int makeArrayConsecutive2(int[] statues) {

    Arrays.sort(statues);
    int count = 0;
   
    for(int i = 0; i<statues.length - 1; i++){
        count+= (statues[i+1] - statues[i]) - 1;
    }
    return count;
}c 